elide-core/src/main/java/com/yahoo/elide/core/exceptions/HttpStatusException.java

```diff
@@ -37,10 +38,12 @@ public HttpStatusException(String message, Throwable cause) {
         super(message, cause, true, log.isTraceEnabled());
     }
 
-    public Pair<Integer, JsonNode> getErrorResponse() {
+    public Pair<Integer, JsonNode> getErrorResponse(SecurityMode securityMode) {
+        boolean useProvidedMessage = getMessage() == null || securityMode == SecurityMode.VERBOSE_ERRORS;
```

Comment:
I don't think `getMessage() == null` is what we want here. This actually modifies the semantics of what we had. Anywhere we had a valid message before will now be swallowed since we determined whether or not to print a message iff `getMessage() != null`.